Products/CMFPlone/resources/browser/resource.py

```diff
@@ -187,6 +187,7 @@ def check_dependencies(bundle_name, depends, bundles):
                     async_=record.load_async or None,
                     defer=record.load_defer or None,
                     integrity=not external,
+                    **{'data-bundle': name},
```

Comment:
The following looks simpler and should be the same, right? Or am I missing something?

```suggestion
                    data-bundle=name,
```

Same a few lines further.

Reply:
Ah, I see, `data-bundle` is probably illegal as parameter and you would get a SyntaxError. Never mind.

Reply:
Exactly, In his way, I contributed to a new feature in webresource package v1.2 to solve this inconvenient. Now accepts more parameters and could be pass with a dict of attributes.

https://github.com/conestack/webresource/blob/master/webresource/_api.py#L134